### DISCUSSION TITLE
New but getting there!

### DISCUSSION THREAD

**Discussion by strebor76** [Bodmer/TFT_eSPI#1517] [New processor/display support or capability request]
Hi all.

Hope everyone had a safe new years eve?

It has taken me time, but thanks to all the resources I have managed to get the cats eye (amongst the others) to display on the 240x240 GC9A01 round lcd i have. But, i must be missing the last little part, because i can only get it to display in the top quarter of the screen. I can move the image to the right (single eye), and have it centred along the x axis, but how (if possible) can i make it centred in the actual screen, or even better, make the animation full screen?
Running on an ESP32.

Thanks all.

**Comment by Bodmer** [comment]
Change the 0 (it is the y position) in this line to 56 and the eye should be centred.

https://github.com/Bodmer/TFT_eSPI/blob/master/examples/Generic/Animated_Eyes_1/eye_functions.ino#L114

The sketch does not allow the eye to be scaled. It was written for 128x128 displays which have a larger pixel size, so then the eye appers bigger.
